README.md

```diff
@@ -0,0 +1,224 @@
+SFTP access by default, so SSHFS is very simple to use--there's nothing to do on the server-side.
+
+
+## Development Status
+
+SSHFS is shipped by all major Linux distributions and has been in production use across a wide range
+of systems for many years. However, at present SSHFS does not have any active, regular contributors,
+and there are a number of known issues (see all issues and bugs in the [SSHFS issue
+tracker](https://github.com/libfuse/sshfs/issues)). The current maintainer continues to apply pull
+requests and makes regular releases, but unfortunately has no capacity to do any development beyond
+addressing high-impact issues. _When reporting bugs, please understand that unless you are including
+a pull request or are reporting a critical issue, you will probably not get a response._
+
+
+## How to use
+
+Once `sshfs` is installed (see the "[Build and Install](#build)" section below) running it is very
+simple.
+
+```bash
+sshfs -h         # help menu
+sshfs --version  # see version 
+```
+
+### 1. To mount a remote filesystem with `sshfs`:
+
+General command syntax:
+```bash
+sshfs [user@]hostname_or_ip:[directory] mountpoint
+```
+
+For example, to connect to the username "john" on a host at local IP `10.0.0.1`, mounting the host's
+root directory (`/`) in your local `~/mnt/sshfs` directory, you would do the following:
+
+```bash
+mkdir -p ~/mnt/sshfs  # create the ~/mnt/sshfs directory, if it doesn't already exist 
+sshfs john@10.0.0.1:/ ~/mnt/sshfs
+```
+
+### 2. To unmount the remote filesystem:
+
+General syntax:
+```bash
+# For most systems, including Ubuntu
+fusermount -u mountpoint
+
+# For BSD and MacOS, and also works fine on Ubuntu
+umount mountpoint
+```
+
+For the example above:
+```bash
+# For most systems, including Ubuntu
+fusermount -u ~/mnt/sshfs
+
+# For BSD and MacOS, and also works fine on Ubuntu
+umount ~/mnt/sshfs
+```
+
+### 3. Notes:
+1. It is recommended to run SSHFS as a regular user (NOT as root). For this to work, the mount point
+   must be owned by the user. Therefore, mounting into a `mount` or `mnt` directory you create 
+   inside your home directory is a good practice.  
+1. If the username is omitted, SSHFS will use the local username. 
+1. If the directory is omitted, SSHFS will mount the (remote) home directory.
+1. If you need to enter a password, SSHFS will ask for it (actually, it just runs `ssh` which asks
+   for the password if needed).
+
+Also, many `ssh` options can be specified. See the manual pages for _sftp(1)_ (`man 1 sftp`) and
+_ssh_config(5)_ (`man 5 ssh_config`). The remote port number (`-oport=PORT`) is one of the many
+`ssh` options which works with `sshfs`.
+
+
+<a id="build"></a>
+## Build and Install
+
+### 1. General overview
+
+First, download the latest SSHFS release from https://github.com/libfuse/sshfs/releases. On Linux
+and BSD (as opposed to MacOS), you will also need to install [libfuse][libfuse] 3.1.0 or newer. On
```

Comment:
On MacOS, you need to install OSXFUSE I believe.

Reply:
This is stated just one line below.